tasks/setup-Debian.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensure that gcc is available.
+  apt:
+    name: gcc
+    state: present
+    update_cache: yes
+  become: yes
+
+- name: Check if homebrew grandparent directory is already in place.
```

Comment:
Technically 'parent' (not grandparent) according to the registered var name?

Reply:
More context:

```yaml
- name: Check if homebrew grandparent directory is already in place.
  stat:
    path: "{{ homebrew_prefix | dirname }}"
  register: homebrew_prefix_parent
  check_mode: no
```

In `defaults/main.yml` we have

```yaml
homebrew_install_path: "{{ homebrew_prefix }}/Homebrew"
```

My thinking is that

- `homebrew_install_path` is the "homebrew directory" (more precisely the "homebrew install directory")
- `homebrew_prefix` is the "homebrew parent directory"
- `homebrew_prefix | dirname` is the "homebrew grandparent directory"

so I do not think we want to replace "grandparent" with "parent" here. If you think this is confusing, then we can come up with something better.

Or maybe I should just change `homebrew_prefix_parent` to `homebrew_grandparent`?